lib/ellis.rb

```diff
@@ -54,9 +54,14 @@ def self.destroy_leaked_numbers(domain)
                         email: EMAIL,
                         password: "Please enter your details")
     cookie = r.cookies
-    r = RestClient::Request.execute(method: :get,
-                                    url: ellis_url(domain, "accounts/#{EMAIL}/numbers"),
-                                    cookies: cookie)
+    begin
+      r = RestClient::Request.execute(method: :get,
+                                      url: ellis_url(domain, "accounts/#{EMAIL}/numbers"),
+                                      cookies: cookie)
+    rescue RestClient::Exception => e
+      puts "Listing existing numbers failed with HTTP code #{e.http_code}, body #{e.http_body}"
```

Comment:
I think the detail of the body is useful, especially after my other PR adding the upstream URL - being able to see at a glance that this live test error is because a request to Homestead's /associated_private_ids URL returned 404 feels like a good thing. I will limit it to just the detail section, and put it on a newline, though.